deploy/addons/registry/registry-svc.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    kubernetes.io/minikube-addons: registry
```

Comment:
The service also needs the addon-manager label `addonmanager.kubernetes.io/mode: Reconcile`.  Unfortunately the addon-manager will ignore any resource without this label.